packages/components/src/buttons/docs/migration-notes-button-group.md

```diff
@@ -1,2 +1,2 @@
-- `fluid` is renamed to `isFluid`.
-- `disabled` is renamed to `isDisabled`.
\ No newline at end of file
+- `fluid` has been renamed `isFluid`.
```

Comment:
I'm not sure anymore if we want to state each boolean. I know later on we just say that the booleans are now prefixed with `is`, or something like that,

Reply:
This might be easier though from a DX point to be able to do a find in the page to figure out why the migration is not working. searching disabled would find the migration note, where otherwise it would take more time to understand that disabled is now prefixed by `is` since it's a boolean.

Reply:
possibly. I don't mind either way.